builder/azure/arm/artifact_test.go

```diff
@@ -42,7 +42,7 @@ func TestArtifactIdVHD(t *testing.T) {
 }
 
 func TestArtifactIDManagedImage(t *testing.T) {
```

Comment:
There should be a test case if snapshot is empty.

Reply:
Additional test cases have been added.